docs/charter.md

```diff
@@ -1,28 +1,70 @@
 ICU4X Charter
 ==============
 
-ICU4X is a new project whose objective is to solve the needs of clients who wish to provide client-side i18n for their products in resource-constrained environments.
+The ICU4X project designs and develops a set of modular internationalization components suitable for use on the client-side and in resource-constrained environments.
 
-ICU4X will be built from the start with several key design constraints:
+ICU4X builds on the design and API decisions of [`ICU4C`]/[`ICU4J`] and [`ECMA-402`].
+
+## Design
+
+ICU4X is being built from the start with several key design constraints:
 
 1. Small and modular code.
 2. Pluggable locale data.
 3. Availability and ease of use in multiple programming languages.
 4. Written by i18n experts to encourage best practices.
 
-ICU4X will provide an API surface in the target client-side platforms, including the web platform, iOS, Android, WearOS, WatchOS, Flutter, and Fuchsia, supported in programming languages including Rust, JavaScript, Objective-C, Java, Dart, and C++.
+## Scope
+
+The scope of ICU4X is the set of functionality covered by [`ECMA-402`], with extensions to cover the needs of clients including Gecko and Fuchsia that are consistent with the spirit of [`ECMA-402`].
+
+One of the benchmarks on decisions related to whether a given API belongs to ICU4X will be if it requires Unicode/CLDR data and would benefit from being managed together with other APIs that rely on the same version of Unicode and CLDR data.
```

Comment:
Suggestion: This paragraph seems redundant with the third bullet point.  I would probably remove it.